docs/common/decorators/parameters.md

```diff
@@ -0,0 +1,11 @@
+Parameters are declared using the `#!typescript @Parameter` decorator. 
+The decorated variable names the parameter. If you assign a value to that
+variable, it becomes the parameter's default value. The `#!typescript @Parameter`
+decorator adds additional metadata via a single argument, a JavaScript object
```

Comment:
I had to read this sentence a couple of times. I think the comma becomes a colon it is simpler to understand.